t/e2e/transactions/transactions_test.go

```diff
@@ -40,162 +25,6 @@ type TransactionsTestSuite struct {
 	e2e.BackendTestSuite
 }
 
-func (s *TransactionsTestSuite) TestCallRPCSendTransaction() {
-	CheckTestSkipForNetworks(s.T(), params.MainNetworkID)
-
-	s.StartTestBackend()
-	defer s.StopTestBackend()
-
-	EnsureNodeSync(s.Backend.StatusNode().EnsureSync)
-
-	err := s.Backend.SelectAccount(TestConfig.Account1.Address, TestConfig.Account1.Password)
-	s.NoError(err)
-
-	transactionCompleted := make(chan struct{})
-
-	var signResult sign.Result
-	signal.SetDefaultNodeNotificationHandler(func(rawSignal string) {
-		var sg signal.Envelope
-		err := json.Unmarshal([]byte(rawSignal), &sg)
-		s.NoError(err)
-
-		if sg.Type == signal.EventSignRequestAdded {
-			event := sg.Event.(map[string]interface{})
-			//check for the correct method name
-			method := event["method"].(string)
-			s.Equal(params.SendTransactionMethodName, method)
-
-			txID := event["id"].(string)
-			signResult = s.Backend.ApproveSignRequest(txID, TestConfig.Account1.Password)
-			s.NoError(signResult.Error, "cannot complete queued transaction %s", txID)
-			close(transactionCompleted)
-		}
-	})
-
-	result := s.Backend.CallRPC(`{
```

Comment:
As mentioned in another comment, this test can stay but the error should be returned when calling RPC with `eth_sendTransaction` method.

Reply:
Addressed